frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -167,6 +167,18 @@ public EnhancedGraphTable(Project project) {
     subscribeToMacheteFileChange();
   }
 
+  public void refreshMacheteFile(boolean asynchronous) {
```

Comment:
Is there a case where we'd want to call it async'ly?

Reply:
You're right, there's probably none.